webfront/views/cache.py

```diff
@@ -31,11 +54,9 @@ def get_timeout_from_path(path, endpoint_levels):
         # it doesn't have modifiers
         if len(query.keys()) == 0:
             return SHOULD_NO_CACHE
-        if (  # The only modifier is page_size
-            len(query.keys()) == 1
-            and "page_size" in query
-        ):
-            return SHOULD_NO_CACHE
+        for parameter in no_cache_modifiers:
+            if parameter in query:
+                return SHOULD_NO_CACHE
```

Comment:
Maybe we could check if each query parameter starts with any of `no_cache_modifiers`? 

For instance, let's say that one day we have a flag that indicates whether a member database signature is a candidate to integration. 
We add a modifier to filter by this flag, e.g. `/api/entry/pfam/?candidates=yes`: `ida` is in `candidates`, so any query with `candidates` would never be cached. 
(Yes, it's far fetched :sweat_smile:)

Reply:
The variable `query` is a dictionary, so when using `in` there is checking the whole value and not a substring

Reply:
Hmm... why did I think that `query` was a string here, but  a dictionary when checking `short_life_parameters`? :man_facepalming:
Sorry about that, no need to change anything, then.